CHANGELOG.rst

```diff
@@ -1,6 +1,28 @@
 Changelog
 =========
 
+
+1.15.1 (2023-03-09)
+-------------------
+
+Bug Fixes
+"""""""""
+
+- Support retriveing extra fields which do not have extra prefix set in ``AzureDataFactoryHookAsync``.
```

Comment:
this needs to be changed similar to the airflow PR description

Reply:
Changed